src/js/DOMTabela.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-unused-vars */
+const domTabela = document.getElementById('domTabela')
+const redovi = document.getElementById('redovi')
+const kolone = document.getElementById('kolone')
+const dugme = document.getElementById('dugme')
+
+for (let i = 0; i < 21 + 1; i++) {
+  redovi.innerHTML += `<option>${i}</option>`
+  kolone.innerHTML += `<option>${i}</option>`
+}
+
+function generisiTabelu (brojRedova, brojKolona) {
+  let tabela = '<table>'
+  for (let i = 0; i < brojRedova; i++) {
+    tabela += '<tr>'
+    for (let j = 0; j < brojKolona; j++) {
+      tabela += `<td>${j}</td>`
+    }
+    tabela += '</tr>'
+  }
+  tabela += '</table>'
+  domTabela.innerHTML = tabela
```

Comment:
Kad funkcija može da se napiše tako da prihvata parametre i sa return vraća vrednost a ne radi ništa usput, to je bolje nego kad se kombinuje to dvoje.
`domTabela.innerHTML = tabela` je "side effect" i ne bi trebal da stoji ovde. Ova funkcija i generiše tabelu i ispisuje je u html, kad već može neka radi samo jedno od ta 2.